src/View/OutputView.js

```diff
@@ -0,0 +1,123 @@
+  },
+
+  printBeforeDiscountTotalOrderAmount(totalPrice) {
+    Console.print(PRINT_MESSAGE.beforeDiscountTotalOrderAmount);
+    Console.print(`${totalPrice.toLocaleString()}원`);
+  },
+
+  printGiveawayMenu(beforeDiscountTotalPrice) {
+    Console.print(PRINT_MESSAGE.giveawayMenu);
+    Console.print(
+      beforeDiscountTotalPrice >= GIFT_EVENT_STANDARD
+        ? PRINT_MESSAGE.giveChampagne
+        : PRINT_MESSAGE.doesNotExist,
+    );
+  },
+
+  printBenefitDetails(discountCalculator, beforeDiscountTotalPrice) {
+    Console.print(PRINT_MESSAGE.benefitDetails);
+
+    if (discountCalculator.totalBenefitPrice === 0) {
+      Console.print(PRINT_MESSAGE.doesNotExist);
+      return;
+    }
+
+    this.printChristmasDDayDiscount(discountCalculator.christmasDDayDiscount);
+    this.printWeekdayDiscount(discountCalculator.weekdayDiscount);
+    this.printWeekendDiscount(discountCalculator.weekendDiscount);
+    this.printSpecialDiscount(discountCalculator.specialDiscount);
+    this.printGiftMenuPrice(beforeDiscountTotalPrice);
+  },
+
+  printChristmasDDayDiscount(christmasDDayDiscount) {
+    if (christmasDDayDiscount > 0) {
+      Console.print(
+        `${BENEFIT_MESSAGE.christmasDDay} -${christmasDDayDiscount.toLocaleString()}원`,
+      );
+    }
+  },
+
+  printWeekdayDiscount(weekdayDiscount) {
+    if (weekdayDiscount > 0) {
+      Console.print(`${BENEFIT_MESSAGE.weekday} -${weekdayDiscount.toLocaleString()}원`);
+    }
+  },
+
+  printWeekendDiscount(weekendDiscount) {
+    if (weekendDiscount > 0) {
+      Console.print(`${BENEFIT_MESSAGE.weekend} -${weekendDiscount.toLocaleString()}원`);
+    }
+  },
+
+  printSpecialDiscount(specialDiscount) {
+    if (specialDiscount > 0) {
+      Console.print(`${BENEFIT_MESSAGE.special} -${specialDiscount.toLocaleString()}원`);
+    }
+  },
+
+  printGiftMenuPrice(beforeDiscountTotalPrice) {
+    if (beforeDiscountTotalPrice >= GIFT_EVENT_STANDARD) {
+      Console.print(`${BENEFIT_MESSAGE.giveEvent} -${MENU_PRICES.샴페인.toLocaleString()}원`);
+    }
+  },
+
+  printTotalBenefitAmount(totalBenefitPrice) {
+    Console.print(PRINT_MESSAGE.totalBenefitAmount);
+    Console.print(
+      totalBenefitPrice === 0
+        ? `${totalBenefitPrice}원`
+        : `-${totalBenefitPrice.toLocaleString()}원`,
+    );
+  },
+
+  printAfterDiscountEstimatedPaymentAmount(expectedPrice) {
+    Console.print(PRINT_MESSAGE.AfterDiscountEstimatedPaymentAmount);
+    Console.print(`${expectedPrice.toLocaleString()}원`);
+  },
+
+  printDecemberEventBadge(totalBenefitPrice) {
+    Console.print(PRINT_MESSAGE.decemberEventBadge);
+
+    const events = [
+      { threshold: BADGE_STANDARD_PRICE.santaBadge, badge: EVENT_BADGE.santa },
+      { threshold: BADGE_STANDARD_PRICE.treeBadge, badge: EVENT_BADGE.tree },
+      { threshold: BADGE_STANDARD_PRICE.starBadge, badge: EVENT_BADGE.star },
+    ];
+
+    const matchingEvent = events.find((event) => totalBenefitPrice >= event.threshold);
```

Comment:
구조분해할당으로 단축 가능해보입니다!

```suggestion
    const matchingEvent = events.find(({ threshold }) => totalBenefitPrice >= threshold);
```